content/en/news/releases/1.6.x/announcing-1.6.7/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Announcing Istio 1.6.7
+linktitle: 1.6.7
+subtitle: Patch Release
+description: Istio 1.6.7 patch release.
+publishdate: 2020-07-30
+release: 1.6.7
+aliases:
+    - /news/announcing-1.6.7
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.6 and Istio 1.6.7.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue which prevented endpoints that are not associated with pods from working. ([Issue #25974](https://github.com/istio/istio/issues/25974))
```

Comment:
Do we want to remove the `that are` from this as well (we did in the 1.6.6 warning)?

Reply:
Done. Thanks.